pageserver/src/http/routes.rs

```diff
@@ -1875,14 +1875,14 @@ async fn timeline_detach_ancestor_handler(
         let ctx = RequestContext::new(TaskKind::DetachAncestor, DownloadBehavior::Download);
         let ctx = &ctx;
 
+        // FIXME: why is there no conversion?
```

Comment:
this comment is a bit mysterious?  Not sure what conversion it's talking about

Reply:
Conversion from the get active timeline to apierror, I was supposed to look it up.